ui/src/textbox.tsx

```diff
@@ -97,7 +97,12 @@ export const
       : (
         <Fluent.TextField
           data-test={m.name}
-          styles={m.multiline && m.height ? { fieldGroup: { height: m.height } } : undefined}
+          styles={m.multiline && m.height
+            ? {
+              field: { height: m.height },
+              fieldGroup: { border: 'none', borderRadius: 0, outline: '1px solid' }
```

Comment:
The default `min-height` is set to `60px` so let's override that as well to give user the behavior according to spec - if height is specified, use that height.
```suggestion
              fieldGroup: { border: 'none', borderRadius: 0, outline: '1px solid', minHeght: m.height }
```

Reply:
Good point @mturoci , thanks.